security/pkg/probe/controller.go

```diff
@@ -35,6 +35,7 @@ const (
 	// LivenessProbeClientIdentity is the default identity for the liveness probe check
 	LivenessProbeClientIdentity   = "k8s.cluster.local"
 	probeCheckRequestedTTLMinutes = 60
+	logEveryNChecks               = 20
```

Comment:
Seems like an arbitrary choice. May be add comment why 20. Do we need this configurable, btw?

Reply:
We don't support customers configuring for simplicity of our config arguments. We can support if customers really ask for it.